### DISCUSSION TITLE
Forslag til nytt ikon

### DISCUSSION THREAD

**Discussion by BeoPeter** [navikt/Designsystemet#3446] [Ikoner]
Veiledere i Syfo har behov for å se at lege har gjort en oppdatering i en sykemelding. Det brukes i dag et feilaktig ikon som ellers ville blitt brukt til menyer, som veiledere har lært seg at betyr "oppdatert et eller annet sted i denne sykemeldingen så nå må jeg klikke inn og lete". 
Forslaget er å lage et ikon som indikerer en oppdatering i et form, sykemelding eller et dokument veileder allerede har vurdert, og burde nå gå inn å gjøre en ny vurdering av. 

Det er mange andre feil i grensesnittet vårt, og det er ikke sikkert at et ikon er det som burde legges på for å ordne i det, men jeg tenker at et slikt ikon kan ha et større bruksområde enn oss, nå som det legges opp til en mer dynamisk dialog mellom bruker og veileder.

**Comment by sjur-gr** [comment]
Her ville jeg brukt et ikon som kommuniserer hva det er (skjema, dokument, etc.) og legge på en dott i øverste høyre hjørne. Det er det vanlige mønsteret for å indikere at en ting har endringer / nytt innhold. Vi har ikke indikatoren som en komponent ennå, men dere kan sikkert kopiere den fra dekoratøren (nav.no menyen). [Vi har konseptet i Alpha UI kit](https://www.figma.com/design/Jw3DCrpJgK0uiG7v92Wdnk/Alpha-UI-kit?node-id=571-27889).
